tests/conftest.py

```diff
@@ -138,7 +138,7 @@ def _check_pods_count():
         assert len(pods) == pods_count
 
     error_msg = (
-        "There is less than '{count}' or no pods labeled '{label}' running"
+        "There is less than '{count}' or no pods labeled '{label}' running "
```

Comment:
Message not really accurate: we check for equality so it could fails even if we have more than  `pod_count` (instead of less).

If possible, we could have `Expected {count} pods labeled …, got {real_count}` but I don't know if we have access to `real_count` easily.

Reply:
Yeah, not sure it's easily doable to get the real count, that's why I did that.
But I can change the message to at least be more accurate.

Reply:
done